cmd/up/up.go

```diff
@@ -142,7 +142,7 @@ func Up() *cobra.Command {
 			}
 			os.Setenv(model.OktetoNameEnvVar, oktetoManifest.Name)
 
-			if len(oktetoManifest.Dev) == 0 {
+			if len(oktetoManifest.Dev) == 0 && !upOptions.DockerDesktop {
```

Comment:
@jLopezbarb can we make this more general? We need the same behavior for the vscode extension

Reply:
@rberrelleza this one in particular only applies to the docker extension, and it's only done on this feature branch.
what would you need for the vscode extension?

Reply:
What happens when this feature is merged to main?

Reply:
@jmacelroy we will need to evaluate how to do it. There are two options:
- Move this logic to the Docker Extension itself
- Make this logic more generic and implement the detach flag

Reply:
I think that making a generic option for "don't ask any questions and just assume defaults" is very useful (we need that for the okteto vscode extension for the same reason than docker desktop).  But I think that's a different requirement from a "detached mode". To me, the "ask no questions" mode sounds like a requirement for detached, but they not need to be tightly coupled. You might still want the process at the forefront, just without requiring any input.

Reply:
Makes sense. I would do that on a separate PR and analyze where we are prompting for questions in the current codebase.